sdk/core/Azure.Core/src/Pipeline/HttpPipelineTransportOptions.cs

```diff
@@ -17,7 +17,9 @@ public class HttpPipelineTransportOptions
         /// </summary>
         public HttpPipelineTransportOptions()
         {
+#pragma warning disable CA1416
```

Comment:
Please add a comment describing what the rule is that is being suppressed and, if needed, why.